tutor/src/components/course-calendar/duration.cjsx

```diff
@@ -81,25 +80,17 @@ CourseDuration = React.createClass
       .value()
 
```

Comment:
Hm, I think valueOf returns the # of milliseconds since epoch  http://momentjs.com/docs/#/displaying/unix-timestamp-milliseconds/  Which will most likely be the same number of digits, but won't always be.  Think the case where some of the dates straddle the point where the milliseconds wrap to the next digit.